changelog.txt

```diff
@@ -1,3 +1,8 @@
+Version 1.15.1-hf1
```

Comment:
Hm, this will get merged later to dev and 1.15.2... so this will be accounted for I think?

Reply:
Should it call out that it fixes support for certain ADFS CertAuthorities configurations? Otherwise it seems like a strange reason to hotfix.